command/server/listener.go

```diff
@@ -177,7 +177,12 @@ PASSPHRASECORRECT:
 		if disableClientCerts && requireVerifyCerts {
 			return nil, nil, nil, fmt.Errorf("'tls_disable_client_certs' and 'tls_require_and_verify_client_cert' are mutually exclusive")
 		}
-		tlsConf.ClientAuth = tls.NoClientCert
+		if disableClientCerts {
+			tlsConf.ClientAuth = tls.NoClientCert
+		}
+		if !disableClientCerts {
```

Comment:
This `if` section shouldn't be here. Only the first is needed.

Reply:
If "tls_disable_client_certs" is set to false, then if client provides a certificate shouldn't it be validated.

Reply:
Sorry I overlooked the default value set.